### PR TITLE
Install Generator

### PR DESCRIPTION
**Pull request by samcdavid**
#5

Why:

So that we can run one command to unpack
everything that the CMS needs to run.

This change addresses the need by:

Add gem dependencies to the gemspec file
and adding a rails generator to run the
active admin install.

### CODE REVIEW

**Comment by Mrjaco12** [comment]
:+1:

File: lib/generators/atomic_cms/install/install_generator.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+      def install_active_admin
+        setup_cms_route_engine
+        setup_active_admin
+        initialize_active_admin_assets
+      end
+
+      private
+
+      def initialize_active_admin_assets
```
Comment (bbrock25) [inline lib/generators/atomic_cms/install/install_generator.rb]: @samcdavid what is this calling? I'm not super familiar with generators...

Comment (samcdavid) [inline lib/generators/atomic_cms/install/install_generator.rb]: `lib/generators/atomic_cms/assets/assets_generator.rb`

**Comment by bbrock25** [comment]
sure @samcdavid :+1: if you say so

**Comment by craigbcooper** [comment]
:+1: Cute Pic

**Pull request merged**